pkg/operator/ceph/csi/csi.go

```diff
@@ -366,3 +367,26 @@ func (r *ReconcileCSI) setParams() error {
 
 	return nil
 }
+
+func (r *ReconcileCSI) deleteRookCSICMIfExists() error {
```

Comment:
```suggestion
func (r *ReconcileCSI) deleteRookCSICMIfExists() error {
```

Can we move this to spec.go the file where the other deletion code is already there?

probably after this https://github.com/rook/rook/blob/e53aaf79ebf564ee1de08db38593baa57a512bbd/pkg/operator/ceph/csi/spec.go#L698

Reply:
done